test/EndToEnd/tests/BuildIntegratedTest.ps1

```diff
@@ -111,6 +111,7 @@ function Test-BuildIntegratedUninstallNonExistantPackage {
 }
 
 function Test-BuildIntegratedLockFileIsCreatedOnBuild {
+    [SkipTest('https://github.com/NuGet/Home/issues/12104')]
```

Comment:
I don't think this attibute works without a `param()` line below it. See other (working) examples of skipped e2e tests.

Reply:
Other tests seem to have the param. Rather than waiting for the build to pass or testing manually (which would take long), I'll trust you :D